sdk/tests/deqp/modules/shared/glsLifetimeTests.js

```diff
@@ -992,11 +992,11 @@ glsLifetimeTests.OutputAttachmentTest.prototype.iterate = function() {
     var element = elementType.gen();
 
     attacher.initAttachment(0, element);
+    this.m_outputAttacher.drawAttachment(element, refSurface);
     attacher.attach(element, container);
```

Comment:
in drawAttachment, buffer is bound to ARRAY_BUFFER firstly. After drawing, buffer is unbound from ARRAY_BUFFER.
In attach(elemetn, container), buffer is bound to TRANSFORM_FEEDBACK_BUFFER, but isn't unbound.
So moving drawAttachment before attach will not trigger "A buffer bound to TRANSFORM_FEEDBACK_BUFFER can not be bound to any other targets" error.